contrib/go-redis/redis.v7/option.go

```diff
@@ -11,6 +11,10 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/internal"
 )
 
+const (
```

Comment:
Why did you add this constant? We don't really need a global constant. It's not reused anywhere. Best revert back to how it was...